### DISCUSSION TITLE
Make DisallowTouchThreshold as float

### DISCUSSION THREAD

**Discussion by workgroupengineering** [CommunityToolkit/Maui#1201] [New Feature Discussions]
If you set a TouchEffect.LongPressCommandParameter on a SwipeView and want to cancel the on operation you can use a DisallowTouchThreshold < and this causes problems.

**Comment by TheCodeTraveler** [comment]
Ping @kphillpotts 

(Kym is currently working on TouchEffect)

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
